metricbeat/Makefile

```diff
@@ -22,21 +22,22 @@ include ${ES_BEATS}/libbeat/scripts/Makefile
 # Collects all module dashboards
 .PHONY: kibana
 kibana:
-	-rm -r etc/kibana
+	# To not remove index-pattern as generated by update
+	-rm -r etc/kibana/dashboard etc/kibana/search etc/kibana/visualization
 	mkdir -p etc/kibana
 	-cp -r module/*/_meta/kibana etc/
 
 # Collects all module and metricset fields
 .PHONY: fields
 fields:
 	mkdir -p etc/
-	cat ${ES_BEATS}/metricbeat/etc/_meta/fields_base.yml > etc/fields.yml
-	. ${PYTHON_ENV}/bin/activate; python ${ES_BEATS}/metricbeat/scripts/fields_collector.py >> etc/fields.yml
+	cat ${ES_BEATS}/metricbeat/etc/fields.yml > etc/fields.generated.yml
```

Comment:
fields.yml contains field which are common for all metricbeat events. The reason I called it fields.yml that in filebeat or winlogbeat the file with all the common options is also called `fields.yml`. `fields_common.yml` would also be an option, but my idea was to call all non generated files which contain `fields.yml`, `fields.yml`.